### PR TITLE
Fix issue with map disappearing on mobile view when zooming in/out

### PR DESCRIPTION
**Pull request by karimbounekhla**
+ includes some Linting changes

### CODE REVIEW

File: src/components/Venues/Map.js [hunk 1]
```diff
@@ -389,11 +437,16 @@ export default class Map extends React.Component {
...
-                </p>
-              </ButtonContent>
-            </Button>
+                <p
+                  style={{
+                    margin: '0 0 0 0.5rem'
+                  }}
+                >
+                  {' '}
+                  {formatMessage(messages.locateMeButton)}{' '}
```
Comment (ParsaHonarmand) [inline src/components/Venues/Map.js:446]: Screen Shot 2020-09-19 at 7 41 21 PM
@karimbounekhla Do we need to initialize formatMessage here for it to be recognized?

Comment (QasimMuhmammad) [reply src/components/Venues/Map.js:446]: Did you two figure this error out?

Comment (karimbounekhla) [reply src/components/Venues/Map.js]: I updated the PR, it should work now with no error

**Comment by QasimMuhmammad** [comment]
Can you close this request and open another one with a merge request with develop branch? Also add the ticket number in the beginning please

**Comment by karimbounekhla** [comment]
> Can you close this request and open another one with a merge request with develop branch? Also add the ticket number in the beginning please

@QasimMuhmammad I'll just close the issue, I couldn't reproduce the bug after the latest merge with prod code

**Pull request closed without merging**